.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "sdk"]
+	path = sdk
+	url = https://github.com/eclipse-velocitas/vehicle-app-cpp-sdk.git
```

Comment:
Can we specify pick submodule not by main but by tag? It will resolve issues in the future.

Reply:
Submodules are always fixed to a certain commit which is not visible in the .gitmodules file but inside of the .git/modules/<module_name>/FETCH_HEAD. Modules just indicate which repo URLs to use.